mlflow/fastai.py

```diff
@@ -380,7 +380,9 @@ def load_model(model_uri):
 
 @experimental
 @autologging_integration(FLAVOR_NAME)
-def autolog(log_models=True, disable=False, exclusive=False):  # pylint: disable=unused-argument
+def autolog(
+    log_models=True, disable=False, exclusive=False, disable_for_untested_versions=False
```

Comment:
@smurching @harupy @WeichenXu123 I'd like to rename `disable_for_untested_versions` to `disable_for_unsupported_versions` because, in the future, there may be other reasons that a version is excluded other than it being untested - e.g. in the event of a broken / fundamentally incompatible ML library release.

We can clarify via documentation that "unsupported versions" includes ML library versions with known incompatibilities *and* versions that have not been tested prior to the given MLflow release. We can also recommend that users upgrade their MLflow client if possible in order to use autologging with the latest ML library versions.

Thoughts?

Reply:
+1